app/graphql/server/settings.js

```diff
@@ -1,9 +0,0 @@
-import { settings } from '../../settings';
-
-settings.addGroup('General', function() {
-	this.section('GraphQL API', function() {
```

Comment:
Need a migration to remove these settings

Reply:
removed in f5f2ebc26b4370038a37e6c9433ea699dce238f9